src/AccessibilityInsights.SharedUx/ActionViews/GeneralActionView.xaml

```diff
@@ -70,7 +70,7 @@
             <Label x:Name="lbDelay" Grid.Column="0" VerticalAlignment="Center" Content="{x:Static properties:Resources.lbDelayContent}"/>
             <TextBox x:Name="tbDelay" Grid.Column="1"
                      Margin="5" AutomationProperties.LabeledBy="{Binding ElementName=lbDelay}"
-                     Text="0" PreviewTextInput="tbDelay_PreviewTextInput"/>
+                     Text="{x:Static properties:Resources.Delay_In_Seconds_Default}" PreviewTextInput="tbDelay_PreviewTextInput"/>
```

Comment:
Since this is for actions, would it make sense to name it `Action_Delay_In_Seconds_Default` ?